### DISCUSSION TITLE
🍯 [Appel à contribution] Candidature financement NGI search | Construction consortium

### DISCUSSION THREAD

**Discussion by AbcSxyZ** [Open-Models/Brique#377] [Share an initiative]
**[Deadline candidature : 2/10/23]**
**Non-french speakers are welcome, going english is a goal !**

[Next Generation Internet (NGI) search](https://www.ngisearch.eu/) est un [financement en cascade](https://www.horizon-europe.gouv.fr/le-financement-en-cascade-pour-les-entreprises-29327) de l'Union Européen avec son programme [Horizon Europe](https://fr.wikipedia.org/wiki/Horizon_Europe), une initiative portée par [5 organisations](https://www.ngisearch.eu/view/Main/Contributors).

NGI propose un financement jusqu'à 150 000 € pour des projets avec des technologies ouvertes exclusivement sur la recherche d'information sur Internet et le web. Orienté surtout logiciel (open source + IA) mais ouverts aux communs numériques et donc l'open education.

**Voir page de postulation (+ guide) :** https://ngi-search-3rd-open-call.fundingbox.com/
**Préparation candidature :** https://mensuel.framapad.org/p/candidature-ngi-search?lang=en

*« L'objectif de cet appel ouvert est de financer et de soutenir des projets qui développeront des technologies et des solutions axées sur les activités liées à l'internet et au web, ainsi que sur la protection de la vie privée et la confiance des utilisateurs qui recherchent et découvrent des informations et des ressources sur Internet/le web. »*

La [brique de connaissances](https://open-models.org/) et le travail d'éducation aux modèles ouverts semble pouvoir bien correspondre et être éligible, cela pourrait être une bonne occasion d'avoir du financement pour du travail réalisé dans ce cadre. J'ai eu une présentation, j'ai demandé si l'open education était admis, visiblement pas une question courante mais l'intervenant, sans savoir le projet, a dit ne pas voir de problème tant que ça rentre dans l'objet de NGI, citant par exemple l'éducation à l'open source qui pourrait être intéressant 😉

J'ai l'intention de candidater (on peut le faire en tant que particulier, jusqu'à 50k/personne), mais je vais voir si cela pourrait intéresser d'autres personnes pour essayer de nourrir davantage l'écosystème comme l'on peut postuler en consortium.

Cette discussion est pour préparer la postulation si d'autres personnes venaient à se joindre, **postuler en consortium renforcerait d'autant plus la candidature.**

## Objectifs de travail

Le financement se base sur un plan de contribution, où l'on définit ses contributions au projet (+ leur valeur) et que l'on divise en 3 étapes.

 ⚠️ **Le financement est réalisé à la fin de chaque étape lorsque les contributions sont terminées, le travail est payé après.** ⚠️

![Screenshot from 2023-09-13 18-58-07](https://github.com/Open-Models/Brique/assets/34010605/c281b36e-75ec-4aa5-a442-20632768be69)
 
Il faut donc définir à l'avance (et faire) un travail en lien avec la brique de connaissances et l'éducation aux modèles ouverts pour envisager recevoir un financement, plan de contribution qui se définit par acteur du consortium.

Chacun pourrait avoir des objectifs spécifiques, en lien avec [la feuille de route](https://open-models.org/organisation/feuille-de-route.html) sur les axes à proposer je réfléchis à :
- L'[internationalisation](https://github.com/AbcSxyZ/markdown-translator), aller vers l'anglais en cherchant le multi-langage.
- Migration sur [Quarto](https://quarto.org/)
- Production du contenu de la brique (création nouvelles pages ou améliorations). Une petite base propre avec les principaux modèles ouverts ?
- Tout le travail de communication/fédération acteurs, sur les médias sociaux voir des interventions d'évènements.
- Chercher à implémenter la brique/éducation aux modèles ouverts dans une(/des) organisation(s) (université, asso, entreprise tech, tiers-lieu...)
- Formation de personnes, en lien avec l'écosystème de formateurs.
- Partenariats ?
- ... ?

À définir plus précisément sur ce qui serait pertinent à avancer, je me placerai sur un certain nombre de ses aspects, à voir ce que d'autres personnes souhaiteraient faire en fonction de leur contexte. Il y a une importante liberté de choix.

Je me dis qu'une des propositions pourrait être de contribuer à leur programme pour former les candidats de NGI search en cherchant à faire avancer l'éducation aux modèles ouverts dans les 2 universités qui gèrent le fond, cela leur prouverait qu'il y a un "marché" s'ils sont eux-mêmes intéressés !

Comme le paiement est postérieur, ça me semble difficile d'envisager des actions qui engagent des fonds (exemple: payer 5 000 pour un développeur/chercheur), à titre personnel c'est surtout du temps de travail qui aurait été fait, pour le moment bénévole, qui devient un bonus pour imaginer à chaque étape injecter des fonds pour la suivante. Des choses à discuter.

## Composition d'une candidature

**Il est possible de candidater en consortium** jusqu'à 3 acteurs européens, personnes physiques ou morales, qui se répartissent jusqu'à 150k sur un projet unique.

Comme je candidate en personne physique, on pourrait avoir jusqu'à 2 acteurs supplémentaires qui se joignent. La volonté est de permettre à d'autre un travail financé sur les modèles ouverts pour leur permettre de consacrer du temps à ces enjeux.

Ouvert à toute personne qui pourrait contribuer et s'inscrire dans ce cadre :
- Formateur/enseignant autour du numérique pour déployer de l'éducation dessus.
- Chercheur ou ingénieur pédagogique pour produire du contenu, enrichir la brique
- Développeur, pour construire la brique ou des outils, pour se consacrer plus à de l'éducation à l'open source (OSPO ?)
- Designer pour avoir des éléments plus beaux voir accessibles.
- Une personne en étude qui souhaite se former et expérimenter ces sujets (avec une aide pour monter en compétence 😉)
- ...

Plein de formes de contributions sont possibles en fonction de tes compétences et envies, je suis preneur de tout ce qui aurait un intérêt. N'hésitez pas à venir échanger ici !

Pour celles et ceux qui seraient volontaires, je vous invite à vous présenter, parler de votre relation aux modèles ouverts (novice ou non ?) et écrire un peu sur ce que vous voudriez faire.

P.S.: Le cas où il y a plus de 2 personnes volontaires n'est pas encore prévu, à discuter en fonction des apports potentiels.
P.P.S: **Pour rappel, il y a jusqu'à 8 mois de travail non payé, le cadre est particulier**, si cela peut éviter le problème précédent.

**Comment by AbcSxyZ** [comment]
Questions à éclaircir :
- Les acteurs du consortium sont-ils financés individuellement par rapport à leurs étapes ? Au moins dans le cas où l'un s'arrête en cours.
- Chaque acteur du consortium est rémunéré directement ?
- Accompagnement dans la sécurité ?

**Reply by AbcSxyZ** [reply]
### Les acteurs du consortium sont-ils financés individuellement par rapport à leurs étapes ? Au moins dans le cas où l'un s'arrête en cours.

*« Il y a une question à laquelle je peux répondre : en cas de consortium, oui, les acteurs sont rémunérés différemment (selon le plan défini au démarrage du projet : à chaque milestone validé, les acteurs ont chacun droit à la part de financement déclarée dans le plan).
Et je ne sais pas ce qu'il en est si un acteur disparaît en cours de projet, je pourrai poser la question, n'hésite pas aussi à faire un mail. Sachant que le but... c'est quand même que çà ne se produise pas, ce serait un très mauvais signal quant au sérieux du projet ! »*

> il faudrait donc que les étapes de chacun soient effectuées pour finir le milestone. Tout le consortium aurait donc la somme débloquée indépendamment mais au même moment, c'est bien ça ?

*« oui, c'est ça »*

**Question sans réponse : Que se passe-t-il si un acteur disparait ?**

**Comment by AbcSxyZ** [comment]
## Proposition mission d'étude : mise en place de formation aux modèles ouverts

**Ceci est une proposition initiallement destinée à l'[IRIIG](https://www.iriig.com/), une école dans l'entrepreneuriat, mais qui reste ouverte à d'autres organisations dans l'enseignement et la recherche. Elle fait suite à une communication sur leurs [missions étudiantes](https://www.linkedin.com/posts/iriig_mission-de-conseil-ugcPost-7105822971998429184-h7fi) qu'on pourrait chercher à inscrire dans cette candidature au financement NGI search [voir début de discussion ci-dessus].**

Le projet de base travaillé ici est une [brique de connaissances sur les modèles ouverts](https://open-models.org/), une ressource ouverte pour chercher à maîtriser l'open science, l'open education, l'open (source) software, l'open hardware... Toutes ces dynamiques de partage et de collaboration autour de l'information et de la connaissance.

La brique est un outil pour chercher à nourrir une culture générale sur ces fondamentaux du numérique et aider à leur démocratisation, un outil d'éducation aux modèles ouverts.

À discuter sur le cadre d'une éventuelle mission, mais la proposition de base pourrait être la suivante : **intégrer au sein de votre propre établissement de la formation aux modèles ouverts.**

Vous faire travailler à votre propre implémentation d'un enseignement aux modèles ouverts autant en s'appuyant sur la brique de connaissances que sur d'autres ressources complémentaires, ressources qui demandent à être améliorée progressivement, voir bien entendu en produisant par vos propres moyens ce que vous jugerez utile à vos besoins. Une réflexion est en cours sur la construction d'un environnement pédagogique pour penser une ingénieurie pédagogique favorisant une transmission lors d'ateliers : https://zenodo.org/record/8021779

Le tout en échangeant et en travaillant ensemble pour réfléchir à structurer cela, c'est un vrai travail exploratoire d'implémentation au sein d'une structure d'enseignement (supérieur ?) d'une éducation aux modèles ouverts.

C'est d'abord un moyen d'intégrer au sein de votre organisation, de vos cursus et plus globalement de l'écosystème autour de vous des connaissances plus abouties sur les modèles ouverts, tout en étant un moyen pour vos étudiants de mieux maîtriser ces compétences numériques autant pour leurs propres projets que pour leur vie personelle et profesionnelle au cours d'un projet pédagogique (et entrepreneuriale) concret au bénéfice de votre structure.

Avec en retour une relative contribution au bénéfice de la brique de connaissances et de l'écosystème globale des modèles ouverts.

Le financement NGI search est géré par 2 universités, Aarhus (Danemark) et Murcia (Espagne), la volonté est de leur proposer de travailler sur l'éducation aux modèles ouverts également, permettant d'aller chercher une collaboration entre vous dans une implémentation de l'éducation aux modèles ouverts au sein de vos propres organisations.

Une collaboration susceptible de s'ouvrir à d'autres acteurs. Toute une création de ressources, de méthodologie, de documentation, de connaissances pourrait être transmissible pour aider à l'implémentation de formation aux modèles ouverts au sein d'autres établissements.

Est-ce que faire travailler vos équipes pédagogique et vos étudiants sur l'éducation aux modèles ouverts vous intéresse ?

Dans le cadre du financement de NGI search, un budget pourrait même être débloqué pour votre structure, pour vos étudiants.

**Reply by AbcSxyZ** [reply]
Voir [Digital Research Academy](https://digital-research.academy/).

Edit: proposition envoyée sur leur mailing list.

**Comment by gvlx** [comment]
Tous ça pourrait être lié avec une stratégie OER (Open Education Ressources) dont le but c'est de partager, sans barrières, la connaissance et les expériences, dans un format approprié à l'utilisation dans les cours de gestion d’entreprises, gestion, etc, avec une licence libre (CC0, CC-BY-SA, etc).

**Reply by AbcSxyZ** [reply]
J'ai vu ton message, très ouvert pour discuter si tout cela peut t'intéresser dans le cadre de ton activité et avec [ESOP](https://www.esop.pt/esop) (Associação de Empresas de Software Open Source Portuguesas) !

L'open education est dans tous les cas au centre du projet, le contenu est déjà sous licence ouverte, tout ce que l'on produit grâce au financement NGI doit l'être également.

Cela semble bien impossible de créer une ressource qui répond à tous les besoins, mais la volonté est de faire de la brique de connaissances un commun pédagogique suffisamment générique qui puisse s'utiliser dans une multitude de contextes.

Comme il y a tout un ensemble d'enjeux sur la stratégie business, cela me semble plutôt pertinent dans le cadre de la gestion d'entreprise.

La proposition de mission d'étude faite au [précédent commentaire](https://github.com/Open-Models/Brique/discussions/377#discussioncomment-7025578) est justement destinée à une école dans l'entrepreneuriat ([IRIIG](https://www.iriig.com/)) pour les aider à intégrer plus pleinement l'éducation aux modèles ouverts au sein de leurs formations.

Quel genre d'idée te donne ce travail d'éducation aux modèles ouverts et la production d'un commun pédagogique par rapport à ce que tu fais ? Des projets dans lesquels ça s'inscrit, une volonté de mieux former au logiciel open source quelque part avec ESOP ? Pour savoir un peu ce que t'inspire cette candidature, où on pourrait peut-être avoir l'opportunité de faire de l'éducation aux modèles ouverts dans des universités(/établissements scolaires) avec l'orientation en cours 🤞

Au moins l'éducation au logiciel open source mériterait à être traité plus sérieusement, les coalitions d'acteurs de l'open source comme ESOP (CNLL, RIOS, OSB, OW2...) pourrait avoir un rôle important à jouer.

P.S.: Si tu préfères basculer en anglais, n'hésites pas !

**Reply by AbcSxyZ** [reply]
## Synthèse visio avec @gvlx

Il y a un intérêt de la part de @gvlx sur ces enjeux d'éducation aux modèles ouverts, constat partagé d'un manque de formation dessus. En suivant mes publications sur les réseaux depuis un moment, avec l'appel à contribution pour NGI, la volonté est de donner un peu d'aide pour faire avancer ces enjeux.

Les sujets évoqués où l'on pourrait travailler ensemble : 
  
**1. Travail sur le contenu de la brique**
Compléter et améliorer le contenu, ce qui pourrait être bien pour travailler la partie logicielle.
  
**2. Soutien sur l'internationalisation**
Aide technique dessus, possiblement une aide pour la traduction portugaise. À regarder la stratégie d'internationalisation d'OW2 sur leur manuel [Good Governance Initiative](https://gitlab.ow2.org/ggi/ggi/) + [Weblate](https://weblate.org/en/)
  
**3. Lien avec les standards, définition processus organisationnel**
Travailler sur la formalisation des processus autour du projet et leur diffusion avec des standards ouverts. Voir [BPMN](https://en.wikipedia.org/wiki/Business_Process_Model_and_Notation), [DMN](https://en.wikipedia.org/wiki/Decision_Model_and_Notation), CMMN, BBPN...
  
**4. Mise en relation avec d'autres d'acteurs**
@gvlx est président de ESOP, de possibles mises en relation dans ce cadre, on a parlé de peut-être faire une proposition au [FOSDEM](https://fosdem.org/2024/) [février 2024] si quelque chose d'intéressant était présentable. En lien avec des acteurs dans la gestion d'entreprises, souhaite inviter à plus d'ouverture chez [Associação para a Inovação Organizacional](https://inovacaoorganizacional.pt/).

Volonté de rassembler prochainement des enseignants (dans l'informatique) pour inciter à la création/usage de ressources éducatives ouvertes.
  
**5. Mise en place d'évènement**
Au moins 1 ! L'envie est partagée, à voir comment l'occasion pourrait se présenter avec un soutien d'autres acteurs.

**Reply by AbcSxyZ** [reply]
@gvlx comment s'est passé ta réunion avec Associação para a Inovação Organizacional vendredi dernier ? Ils semblent intéresser pour travailler sur ces enjeux d'ouverture dans leurs réflexions sur l'innovation organisationnelle ?

**Comment by AbcSxyZ** [comment]
Pour garder l'information, il y a le programme [Open Education for a Better World](https://oe4bw.org/) (OE4BW) qui sert à accompagner dans la création d'OER, en lien avec [The University of Nova Gorica](https://ung.si/en/) and UNESCO Chair on Open Technologies for OER and Open Learning.

Une candidature pourrait-être un moyen d'avoir du support (mentoring) dans la création de la brique et un endroit où diffuser la ressource. Programme sans financement : https://oe4bw.miteam.si/asset/tDsvckrL4ZuHJg8Ka

**Deadline :** 9 November 2023 (23:59 CET)

**Comment by AbcSxyZ** [comment]
## Finalization of the application

5 days until the application deadline, the 2 october at 17 p.m. CEST. The application will be filled out here : https://mensuel.framapad.org/p/candidature-ngi-search

The purpose and scope of the project related to the brick of knowledge and open models education will be explained, one main part is to define our contribution roadmap to the brick project and/or education on open models.

Time is short, but we should be able to apply as a consortium ! Here's the basic proposal I propose we work on.

The core of the project is about building the brick of knowledge, a digital common for open models education, and implementing around it open models education for different organizations and people, with a strong focus on higher education and research institutions.

I think it should be mainly about helping organizations to implement their own open models trainings and empowering people with these digital skills.

With the following organisations to try to start with: consortium members (+ network around), Aarhus & Murcia Universities, NGI search (program) + The University of Nova Gorica and UNESCO Chair on Open Technologies for OER and Open Learning through [Open Education for a Better World](https://oe4bw.org/) (OE4BW), etc.

## Potential consortium and members around the project

We may apply with the following consortium *(50k€ per actor)*:
- [Digital Research Academy](https://digital-research.academy/) (DRA)
- [Open Knowledge Association](https://okalab.org/) (OKA) [website down]
- Me

As consortium members, because your activity is already related to open models, the main intent is to support your work in open models education in your own organization, while expecting your support for the project more globally with things related to your activity.

DRA is a nascent network of open science trainers, working on « training for digital research skills » more broadly. While improving open models capabilities of trainers from your network, you could bring us useful open science skills and significant support to work with the scientific community.

OKA is also a new project with the intent to build eco-campuses for study projects (energy, building, alimentation...) supported by a library for their digital commons. A project still in the making with people I met during their studies. It could be a way for you to shape an education axis at OKA and improve your (training) skills while helping in this wider project.

Personally, my work would be about building the brick, from content to internationalisation, doing maintenance burden (work on contribution), training, organisation, communication, networking and these kinds of things.

We will probably find some support from external actors of the consortium. I discussed with @glvx, president of [ESOP](https://esop.pt/) (Associação de Empresas de Software Open Source Portuguesas), who is interested in helping by providing technical, organisational and networking support.

With the intent to federate over time people interested in the brick of knowledge, in open models education !

## Work Plan and Schedule for Application

We need to define a work plan for each member of the consortium—an individual work plan of a collaborative roadmap that must be completed together to receive funding at the end of each milestone.

See Implementation > Work Plan section in the application : https://mensuel.framapad.org/p/candidature-ngi-search

Anything related to open models education you want to work on, inside your organization, around the brick or with the outside world. They're asking to split work and specify how to verify it (maybe we should specify a verification process when not obvious).

> **Example for the Digital Research Academy, split into their 3 steps:**
> Step 1
> \- Implementing open models education in a DRA train-the-trainers curriculum, visible with content creation and workshop
> Step 2
> \- Setup the DRA library for open educational resources
> Step 3
> \- Support open models training at Aarhus University and researchers around DRA, see with workshops

Next to the work plan, there is a « Resources » section to indicate how we will use NGI search funding. Because funding is coming afterwards, not 100% sure about their expectation but I think it could be about explaining it's for living naturally, investing in the work plan and next milestones, investing fully in your project or another initiative...

**For OKA and DRA teams (and me), you need to provide 300 characters for your own work plan + 300 characters on how you expect to use NGI funding.**

A first (short !) proposition for the application, the work plan will be defined more precisely when the program starts (and eventually if the application is accepted 🤞).

**Ideal proposed schedule:**
*Wednesday*: asking work plan from DRA and OKA
*Friday:* end filling all possible parts of application [help and feedback are welcome !]
*Saturday:* finalizing the work plan together with DRA and OKA
*Sunday/Monday, (until the deadline at 17h CEST):* time for application review

Hope everything will be fine for everyone, I'm available for any questions or to organize [an online meeting](https://cal.com/open-models) to end up this application !

I'd also like to suggest that we organise a meeting all together in the near future, it can be great if everyone can get to know each other 🙃

## What could be next ?

It will be just a step to build a better foundation to the brick and implement open models education in first organizations. The aim is to reach as many people as possible who are likely to want to work on these subjects.

Knock on the doors of other higher education institutions. I'm thinking of Nantes University, which has an [important program on openness](https://www.univ-nantes.fr/decouvrir-luniversite/vision-strategie-et-grands-projets/open-education-leducation-ouverte-a-nantes-universite) and a [chair on it](https://chaireunescorelia.univ-nantes.fr/), we may have some contact with institutions in Switzerland also with the [Swiss Digital Skill Academy](https://d-skills.ch/), other entities around each of us.

With the movement in favor of open science, there are different spaces in universities to work on open models education.

It could be some work with actors in education, like the [Fabrique des Communs Pédagogiques](https://fabpeda.org/) [educational commons factory], the [fabriqueREL](https://fabriquerel.org/) or different kinds of open education actors/networks. Schools and teachers are slowly getting involved.

It could also be in relation to the IT sector and open (source) software actors where there is a need for training. There is some (national) coalition of open source organizations like ESOP for Portugal with @glvx, the president of [OW2](https://www.ow2.org/) is also following this educational work. We may reach some people in [Open Source Program Offices](https://ospoglossary.todogroup.org/ospo-definition/), I was speaking to one of them few days ago, he said he wanted to « transpose [y]our (cross-disciplinary) subject to an applied subject (energy) », a project related to [Linux Foundation Energy](https://lfenergy.org/) 🤷

A variety of places where there are things to do related to open models (education) depending on emerging opportunities.

**Reply by AbcSxyZ** [reply]
## Digital Research Academy (DRA) Work Plan Proposal

Stage 1: Onboarding (1 month), up to 20% total budget
● Milestone 1 — participation in Welcome Event, completing the Individual
Attending the Kick Off, Completing the Individual Mentoring Plan and addressing the Ethics requirements produced by the Ethical Review.

**Budget Breakdown-**
7000 EUR for 1 PM for 1 FTE (DE Rate 7000 EUR/Month)
3000 EUR for  travel and/or planning costs for Kickoff Event
Total: 10.000EUR

**DRA tasks-**
Detailed Project Management Planning and administrative set up of consortium
Attendance of Kickoff event (if internal then we could handle the planning as well)


Stage 2: Implementing (8 months), up to 50% total budget
● Milestone 2 — Development of the MVP and Business Plan

**Budget breakdown-**
22.400 EUR for 8 PM for 0.4 FTE
800 EUR project management platform costs (i.e. Jira, Asana, etc. for 11 EUR/mo/user up to 9 users)
1800 EUR for 2x trainer platform testing workshops
Total: 25.000 EUR

**DRA tasks-**
Content development support
User testing of prototypes with DRA trainers
Course/Workshop development to accompany platform

Stage 3: Adopting (3 months), up to 30% total budget
● Milestone 3 — Completion of the Technology adoption Plan
**Budget breakdown-**
8.400 EUR 3PM for 0.4 FTE
5.600 EUR for organization of onsite TTT with deployment of MVP
1000 EUR travel costs
Total: 15.000 EUR 

**DRA tasks-**
Organization of full TTT program
Deployment of course/workshop developed for onboarding of MVP platform

**Reply by AbcSxyZ** [reply]
## Open Knowledge Association (OKA) Work Plan Proposal

**Step 1**
Stage 1 consists of mapping the landscape of information, education and open models through the drafting and publication of a monitoring document.
This document will cover the history of information technology and the impact of the Internet on the consumer/producer relationship.
This paradigm shift favors open models that meet the challenges of a contributive economy.
It will then address the current problems of the Internet (hosting, censorship, etc.) and the need for new business models in the interests of a society oriented towards open models.

Stage 1 of the NGI project involves publishing the article in open access on the knowledge "brick" and on open science platforms such as Zenodo, Arxive, etc., and requesting open peer review.

**Step 2**
Define open standards for how to integrate and implement truly open models, based on the values/conclusion of our paper. Those standards define collaboration and interoperability between different individuals or organizations.

Therefore, OKA aims to be a standardization organization for open models.

In the scope of the NGI project, we aim to start the process of defining those open standards and start building a community with which we can co-create and improve those standards.

**Step 3**
We will co-develop with students, a campus model compatible with the standards defined in step 2.
Students can aplly knowledge trough projects related to sustainable infrastructure.
They can develop, test and implement those projects in order to shape the campus and the standards.
Therefore, step 2 and step 3 are iterative.

The campus will focus on developing tools and educational ressources about open models such as:

Education within the definition of Open and how to apply it (Business models, Community, Contribution, Etc.)
R&D on infrastructure projects in a contributive economy
Learning by projects related to infrastructure and open models
In the framework of the NGI project, we plan to study the feasibility and the requirements to create such a campus.
Those requirements would for example include permissions, building steps and budget, development of a community of contributors (Universities, research center, companies, etc.)

Stage 1: Onboarding (1 month), up to 20% total budget
● Milestone 1 — participation in Welcome Event, completing the Individual
Attending the Kick Off, Completing the Individual Mentoring Plan and addressing the Ethics requirements produced by the Ethical Review.

Budget Breakdown-
8.000 EUR 3 people write the text
2.000 EUR Processes of open science publication
Total: 10.000 EUR

Stage 2: Implementing (8 months), up to 50% total budget
● Milestone 2 — Development of the MVP and Business Plan

Budget breakdown-
5.000 EUR Hosting digital infrastrure (Set up the digital infrastructure for collaboration)
8.000 EUR Study of feasability of the infrastructure 
12.000 EUR Planning physical infrastructure
Total: 25.000 EUR

Provide a physical place for this community 
Planning infrastrucre with the open technologies to test the standards

Stage 3: Adopting (3 months), up to 30% total budget
● Milestone 3 — Completion of the Technology adoption Plan
Budget breakdown-
10.000 EUR Finding human ressources
1.500 EUR Defining student's projects details
3.500 EUR Communication
Total: 15.000 EUR

**Comment by AbcSxyZ** [comment]
The application was sent :tada:: https://github.com/Open-Models/Brique/discussions/381
